pep-0517.txt

```diff
@@ -658,7 +650,7 @@ above, there are a few other differences in this proposal:
   step and the wheel building step. I guess everyone probably will
   agree this is a good idea?
 
-* We call our config file ``pypackage.json`` instead of
+* We call our config file ``ppyproject.toml`` instead of
```

Comment:
With PEP 518 accepted, we can assume this particular difference between the two PEPs will go away (and that's also assuming @rbtcollins decides to continue with PEP 516 - he's changed jobs since he wrote that, and may no longer have the time or inclination to champion it as a competing proposal)